io-manager.js

```diff
@@ -245,12 +245,13 @@ const _updateIo = timeDiff => {
       keysDirection.applyQuaternion(camera.quaternion);
       _updateVertical(keysDirection);
     } else {
+      const isGrounded = localPlayer.characterPhysics.lastGrounded;

```

Comment:
We should not re-use tracking flags like this. If you want to track if the character is grounded, we should have an API for that.

Reply:
[add isGrounded API in game.js](https://github.com/webaverse/app/pull/3543/commits/f7db09a71021d68980c8022850eaf722daf85f3a)